ui/app/common/authentication/authService.js

```diff
@@ -57,19 +57,24 @@
       };
 
       authService.isSuperUser = function () {
-        return authService.userHasRole('system-user');
+        return authService.userHasRole('pnc-users-admin');
       };
 
       /**
        * Verifies the minimum lifespan of the refresh token.
        */
-      authService.verifySsoTokenLifespan = function () {
-        const MIN_REFRESH_TOKEN_EXPIRY_DATE = Date.now() + authConfig.getSsoTokenLifespan();
```

Comment:
No idea what we were doing here and how time and access token lifespan should correlate with refresh token validity